pages/index.js

```diff
@@ -588,9 +589,9 @@ export const getStaticProps = async ({ params }) => {
   const contentFilePath = path.join(CONTENT_PATH, `index.mdx`)
   const source = fs.readFileSync(contentFilePath)
   const { content, data } = matter(source)
-  var client = new Client("azure");
+  var client = new Client("dev");
```

Comment:
what does this change implies?

Reply:
this is why the build is failing actually, it is trying to hit the local encore instance.  this parameter determines which encore instance to hit.